tests/s3/copy_object/test_copy_object_quota.py

```diff
@@ -0,0 +1,733 @@
+            self.buckets_created.append([bkt_name, self.akey, self.skey])
+        self.log.info("Step 13: Upload a simple object obj1 of some random size S in src1")
+        random_size = self.csm_obj.random_gen.randrange(1, test_cfg["max_size"])
+        resp = self.multipart_upload(self.src_bkt, "obj1", akey, skey, random_size)
+        assert_utils.assert_true(resp, "Multipart upload Failed")
+        self.log.info("Step 14:  Perform Copy object operation from bucket 'src1' --> 'dest1'"
+                      " --> 'dest2' --> ... --> 'destN-1' bucket and using different key name for"
+                      " the destination object during every copy operation.")
+        src_bkt = self.src_bkt
+        src_obj = "obj1"
+        for cnt in range(1, (test_cfg["max_objects"])):
+            dest_bkt = "dest" + str(cnt)
+            dest_obj = "obj" + str(cnt+1)
+            resp = s3_misc.copy_object(akey, skey, src_bkt, src_obj,
+                                       dest_bkt, dest_obj)
+            assert_utils.assert_true(resp, f"Copy object Failed for {dest_bkt}/{dest_obj}")
+            src_bkt = dest_bkt
+            src_obj = dest_obj
+        self.log.info("Step 15: Perform Copy object operation again (N)th time within same bucket"
+                      " using same source bucket and object and just changing the key name of the"
+                      " destination object .")
+        try:
+            dest_bkt = "dest" + str(test_cfg["max_objects"])
+            dest_obj = "obj" + str(test_cfg["max_objects"]+1)
+            s3_misc.copy_object(akey, skey, src_bkt, src_obj, dest_bkt, dest_obj)
+        except CTException as error:
+            self.log.info("Expected exception received %s", error)
+            assert_utils.assert_in(errmsg.S3_COPY_OBJECT_QUOTA_ERR, error.message, error)
+        self.log.info("Step 16:Perform Get API to get user and bucket stats and validate the object"
+                      "count and space utilization for user/bucket .")
+        self.log.debug("Perform & Verify GET API to get capacity usage stats")
+        self.verify_user_quota(akey, skey, uid)
+        self.log.info("ENDED:Test copy-object operation across different buckets(multi-hop) for"
+                      " multipart object when max-object limit is set in user level quota")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags('TEST-46138')
+    def test_46138(self):
+        """
+        Test copy-object overwrite operation for multipart object when max-object limit is set
+        in user level quota
+        """
+        self.log.info("STARTED:Test copy-object overwrite operation for multipart object when"
+                      " max-object limit is set in user level quota")
+        test_cfg = self.csm_conf["test_40632"]
+        config_dict = {"enabled": True, "max_size": test_cfg["max_size"],
+                       "max_objects": test_cfg["max_objects"]}
```

Comment:
max objects is not set in csm config file. pls check this for all tests

Reply:
changed test number. Please check